src/main/kotlin/racingcar/data/Car.kt

```diff
@@ -0,0 +1,16 @@
+package racingcar.data
+
+data class Car(
+    val name: String,
+) {
+
+    private var _location: Int = 0
+    val location: Int
+        get() = _location
+
+    fun move() {
+        _location++
+    }
```

Comment:
아래와 같이 수정하면 Test Code 에서 `repeat { car.move() }` 을 `car.move(6)` 으로 수정할 수 있을 것 같습니다!
```suggestion
    fun move(distance: Int = 1) {
        _location += distance
    }
```